modules/phoenix/src/test/ts/atomic/api/split/SplitTest.ts

```diff
@@ -38,7 +38,7 @@ UnitTest.test('api.Split.(split,splitByPair)', () => {
     const actual = Split.splitByPair(input.universe, input.item, start, finish);
     assert.eq(middle, actual.text);
     assert.eq(expected, input.universe.shortlog((item) => {
-      return item.name === 'TEXT_GENE' ? 'text("' + item.text + '")' : item.id;
+      return input.universe.property().isText(item) ? `text("${input.universe.property().getText(item)}")` : item.id;
```

Comment:
```suggestion
      const props = input.universe.property();
      return props.isText(item) ? `text("${props.getText(item)}")` : item.id;
```
nitpick, especially for a test, but this is a little nicer to read to me because ternary +  string template.

Reply:
We also do this a few times in the follow tests... Makes me wonder if we should have a test helper method for it. I don't mind what we do with any of this though.

Reply:
I think if anything we should try to find an alternative method of comparing trees that isn't serialising it to one big string and then `===`

Reply:
I'll make that change though